controllers/helpers_test.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+//nolint:gosec
```

Comment:
Could you instead put this skip statement at line 405? Then the real code in the top half of this file will still be checked by `gosec`.

Reply:
Sure thing - I was thinking this was all test code based on the name, so skipping it all. But in general it is probably better to be more specific anyway. Thanks!